src/SonarScanner.MSBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -190,7 +190,6 @@ private bool FetchArgumentsAndRulesets(ISonarQubeServer server, ProcessedArgs ar
                     if (!activeRules.Any())
                     {
                         this.logger.LogDebug(Resources.RAP_NoActiveRules, plugin.Language);
-                        continue;
```

Comment:
Don't skip adding the analyzer, ruleset and extra files when there is no active rules.